Assets/Scripts/Controllers/AudioManager.cs

```diff
@@ -0,0 +1,130 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEngine;
+
+/// <summary>
+/// The Manager that handles the loading and storing of audio form streamingAssets.
+/// </summary>
+public class AudioManager
+{
+    private static Dictionary<string, AudioClip> audioClips;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AudioManager"/> class.
+    /// </summary>
+    public AudioManager()
+    {
+        audioClips = new Dictionary<string, AudioClip>();
+    }
+
+    /// <summary>
+    /// Creates a human readable string of the audioClips Dictionary.
+    /// Used for debugging.
+    /// </summary>
+    /// <returns>String containing the information of audioClips.</returns>
+    public static string GetAudioDictionaryString()
+    {
+        Dictionary<string, AudioClip> dictionary = audioClips;
+        return "{" + string.Join(",", dictionary.Select(kv => kv.Key + "=" + kv.Value.length).ToArray()) + "}";
+    }
+
+    /// <summary>
+    /// Gets an AudioCLip from the specified category, with the specified name.
+    /// Throws a LogWarning and returns an Error sound if the AudioClip does not exist.
+    /// </summary>
+    /// <param name="categoryName">
+    /// The category that the AudioClip is in. Usually the same as the 
+    /// directory that the audio file was load from.
+    /// </param>
+    /// <param name="audioName">The name of the AudioClip.</param>
+    /// <returns>AudioClip form the specified category with the specified name.</returns>
+    public static AudioClip GetAudio(string categoryName, string audioName)
+    {
+        AudioClip clip = new AudioClip();
+
+        string audioNameAndCategory = categoryName + "/" + audioName + ".ogg";
+
+        if (audioClips.ContainsKey(audioNameAndCategory))
+        {
+            clip = audioClips[audioNameAndCategory];
+        }
+        else
+        {
+            try
+            {
+                Debug.LogWarning("No audio available called: " + audioNameAndCategory);
+                clip = audioClips["Sound/Error.ogg"];
+            }
+            catch
+            {
+                throw new FileNotFoundException("Sound/Error.ogg not found");
+            }
+
+        }
```

Comment:
Stylecop `SA1508 : CSharp.Layout : A closing curly bracket must not be preceded by a blank line. (Assembly-CSharp)`